### PR TITLE
Allow gist navigation

### PR DESCRIPTION
**Pull request by natefaubion**

### CODE REVIEW

**Comment by natefaubion** [comment]
Note this looks for `gist.github.com` urls, not ?gist= urls.

File: client/index.html [hunk 1]
```diff
@@ -184,6 +184,12 @@
         clearInterval(sendSources);
       }, { once: true });
 
+      window.addEventListener("message", function(event) {
+        if (event.data && event.data.gistId && /^[0-9a-f]+$/.test(event.data.gistId)) {
+          window.location.search = "gist=" + event.data.gistId;
```
Comment (hdgarrood) [inline client/index.html]: Do we need to worry about this clobbering other query string parameters? I haven’t worked out what the `session` one does yet.

Comment (natefaubion) [reply client/index.html]: I just went with this because that's how all the current links work.

Comment (natefaubion) [reply client/index.html]: I don't think this is supposed to preserve session, which saves your editor to local state.

Comment (hdgarrood) [reply client/index.html]: Ahh, of course.

**Pull request merged**